test/test_scatter_plot.py

```diff
@@ -0,0 +1,16 @@
+from scatter import make_scatter_plotter, find_json_files
+from ase.io.trajectory import Trajectory
+import sys
+import os
+import yaml 
+
+config_file = open("test/config.yaml")
+parsed_config_file = yaml.load(config_file, Loader=yaml.FullLoader)
+
+out_file_list = find_json_files(parsed_config_file)
```

Comment:
This test should maybe test something.
Could be better to call this function from inside the test function instead.